src/search/merge_and_shrink/merge_scoring_function_total_order.h

```diff
@@ -15,18 +15,20 @@ class RandomNumberGenerator;
 }
 
 namespace merge_and_shrink {
+enum class AtomicTSOrder {
+    REVERSE_LEVEL,
+    LEVEL,
+    RANDOM
+};
+
+enum class ProductTSOrder {
+    OLD_TO_NEW,
+    NEW_TO_OLD,
+    RANDOM
+};
+
```

Comment:
Why are the enums moved outside the class? Was it necessary or just a style choice?